install/scripts/drupal.sh

```diff
@@ -135,17 +103,12 @@ drush en -y search_api
 #cp -a "$DRUPAL_HOME"/sites/all/modules/apachesolr/solr-conf/solr-4.x/. "$SOLR_HOME"/collection1/conf/
 #service tomcat7 restart
 
-#cd "$DRUPAL_HOME/sites/all/modules"
-#git clone https://github.com/Islandora-CLAW/islandora.git
-#drush -y en islandora
-#drush -y en islandora_dc
-#drush -y en islandora_mods
-#drush -y en islandora_basic_image
-#drush -y en islandora_collection
-#drush -y en islandora_apachesolr
-#drush -y en islandora_delete_by_fedora_uri_service
-#drush -y en islandora_medium_size_service
-#drush -y en islandora_tn_service
+cd "$DRUPAL_HOME/modules"
+git clone https://github.com/DiegoPino/claw-jsonld.git
+drush en -y jsonld
+
+ln -s "$HOME_DIR"/islandora .
```

Comment:
Confirming that is needs to be`ln -s "$HOME_DIR"/islandora/islandora` .
Went line by line executing the script.